strings-csharp/HexToByteArray/HexToByteArray/ConvertHex.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace HexToByteArray
+{
+    public class ConvertHex
```

Comment:
Since there is no instance data in this class, the whole thing can be made static

Reply:
Done. Thanks!